config/base/services/authn-webhook/authn-webhook.yaml

```
@@ -0,0 +1,66 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: authn-webhook
+  labels:
+    app.kubernetes.io/name: authn-webhook
+    app.kubernetes.io/part-of: auth-provider-zitadel
+    app.kubernetes.io/managed-by: kustomize
+spec:
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: authn-webhook
+      app.kubernetes.io/part-of: auth-provider-zitadel
+  replicas: 1
+  template:
+    metadata:
+      annotations:
+        kubectl.kubernetes.io/default-container: manager
+      labels:
+        app.kubernetes.io/name: authn-webhook
+        app.kubernetes.io/part-of: auth-provider-zitadel
+    spec:
+      securityContext:
+        runAsNonRoot: true
+        seccompProfile:
+          type: RuntimeDefault
+      containers:
+        - name: authn-webhook
+          image: auth-provider-zitadel:latest
```

We can use the fully qualified image name here.
```suggestion
          image: ghcr.io/datum-cloud/auth-provider-zitadel:latest
```

---

File updated!